test/test_tracker.cpp

```diff
@@ -316,8 +316,11 @@ TORRENT_TEST(extract_peer_missing_port)
 bool connect_alert(libtorrent::alert const* a, tcp::endpoint& ep)
 {
 	if (peer_connect_alert const* pc = alert_cast<peer_connect_alert>(a))
+	{
 		ep = pc->ip;
-	return true;
```

Comment:
ouch

Reply:
Something wrong with it?

Reply:
no, I was just commenting on me writing this bug :P (that you just fixed)